File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        
        String [] str = s.split(" ");
      
        String max = str[0];
        String min = str[0];
        
        for(String n : str){
            if (Integer.parseInt(n)>Integer.parseInt(max)){
                max=n;
            }
            if (Integer.parseInt(n)<Integer.parseInt(min)){
                min=n;
            }
        }

        String answer = min+" "+max;
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    numbers = list(map(int, s.split()))
    
    # 최소값과 최대값 찾기
    min_num = min(numbers)
    max_num = max(numbers)
    
    # 최소값과 최대값을 문자열로 반환
    answer = f"{min_num} {max_num}"
    return answer